src/api/app/models/event_subscription/find_for_event.rb

```diff
@@ -78,7 +78,10 @@ def expand_receivers(receivers, channel)
 
     def expand_receivers_for_groups(_new_receivers, receiver, channel)
       # We don't subscribe Groups so we have to get the group's users to get the subscriptions
-      return receiver.users if event.instance_of?(Event::RelationshipCreate) || event.instance_of?(Event::RelationshipDelete)
+      if event.instance_of?(Event::RelationshipCreate) || event.instance_of?(Event::RelationshipDelete)
+        return receiver.email_users if channel == :instant_email
```

Comment:
Once the `rss` channel is added here, this would be a use for a `case` statement instead of 3 `if` statements.